projects/vdk-plugins/vdk-meta-jobs/README.md

```diff
@@ -58,84 +58,84 @@ In this example what happens is
 ```python
 
 from vdk.api.job_input import IJobInput
-from vdk.plugin.meta_jobs.meta_job_runner import MetaJobInput
+from vdk.plugin.dags.dag_runner import DAGInput
 
 JOBS_RUN_ORDER = [
     {
         "job_name": "job1",
         "team_name": "team-awesome",
-        "fail_meta_job_on_error": True,
+        "fail_dag_on_error": True,
         "depends_on": []
     },
 
     {
         "job_name": "job2",
         "team_name": "team-awesome",
-        "fail_meta_job_on_error": True,
+        "fail_dag_on_error": True,
         "depends_on": ["job1"]
     },
     {
         "job_name": "job3",
         "team_name": "team-awesome",
-        "fail_meta_job_on_error": True,
+        "fail_dag_on_error": True,
         "depends_on": ["job1"]
     },
     {
         "job_name": "job4",
         "team_name": "team-awesome",
-        "fail_meta_job_on_error": True,
+        "fail_dag_on_error": True,
         "depends_on": ["job1"]
     },
 
     {
         "job_name": "job5",
         "team_name": "team-awesome",
-        "fail_meta_job_on_error": True,
+        "fail_dag_on_error": True,
         "depends_on": ["job3"]
     },
     {
         "job_name": "job6",
         "team_name": "team-awesome",
-        "fail_meta_job_on_error": True,
+        "fail_dag_on_error": True,
         "depends_on": ["job3"]
     },
 ]
 
 
 def run(job_input: IJobInput) - > None:
-    MetaJobInput().run_meta_job(JOBS_RUN_ORDER)
+    DAGInput().run_dag(JOBS_RUN_ORDER)
 ```
 
 ### FAQ
 
 
-**Q: Will the metajob retry on Platform Error?**<br>
+**Q: Will the DAG retry on Platform Error?**<br>
```

Comment:
nit: Will the DAG retry on data jobs composed, in case of a Platform Error during their execution?
My impression is, that in case a composite job fails then it will be retried. 

(nit: What happens - in case the dag itself fails, for example exit 1; after composite data jobs were all successfully executed?)

Reply:
If the DAG fails after the `DAGInput().run_dag()` call finishes, then all the component jobs complete successfully, and the DAG job itself is marked as failed.